coldfront/core/user/views.py

```diff
@@ -95,6 +95,13 @@ def get_context_data(self, viewed_username=None, **kwargs):
 class UserProfileUpdate(TemplateView):
     template_name = 'user/user_profile_update.html'
 
+    def test_func(self, request, *args, viewed_username=None, **kwargs):
```

Comment:
This method doesn't get invoked unless the class inherits from `UserPassesTestMixin`. We don't actually need it, since this view only ever allows editing of the requesting user's information.